src/main/java/net/sf/jabref/importer/fetcher/IEEEXploreFetcher.java

```diff
@@ -377,6 +407,8 @@ private BibtexEntry cleanup(BibtexEntry entry) {
                 fullName = fullName.replaceAll(" on", " ").replace("  ", " ");
             }
 
+            Pattern publicationPattern = Pattern.compile("(.*), \\d*\\.*\\s?(.*)");
```

Comment:
Pattern should be private final static fields because it is very expensive to compile them. And in this case, it would have to be compiled for each fetch.